model/lib/extensions/list_email_address_extension.dart

```diff
@@ -27,6 +27,18 @@ extension SetEmailAddressExtension on Set<EmailAddress>? {
     return listEmail.isNotEmpty ? listEmail.join(', ') : '';
   }
 
+  String asFullHtmlString(String separator) {
+    if (this?.isNotEmpty != true) return '';
+
+    final listEmail = this
+      !.map((emailAddress) => emailAddress.asFullHtmlString())
+      .toList();
+
+    return listEmail.isNotEmpty ? listEmail.join(separator) : '';
+  }
+
+  String asFullHtmlStringUseCommaSeparator() => asFullHtmlString(', ');
```

Comment:
if list have length = 1, we don't need the separator.

Reply:
The `join` function takes care of this.

<img width="644" alt="Screenshot 2024-12-18 at 02 02 00" src="https://github.com/user-attachments/assets/c482b5fd-8d53-47d0-8bd2-40450c3e8acd" />